requirements/edx/pre.txt

```diff
@@ -4,15 +4,14 @@
 #   * @edx/ospr - to check licensing
 #   * @edx/devops - to check system requirements
 
-# Use a modern setuptools instead of distribute
-setuptools==18.0.1
+# Packaging pre-requisites.
+pyparsing==2.0.7
+setuptools==34.0.2
```

Comment:
setuptools also depends on `appdirs`, `packaging`, and `six`; should those be pinned here too?